src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -5296,7 +5308,7 @@ export class StudyViewPageStore {
 
     public async getSurvivalDownloadData(chartMeta: ChartMeta) {
```

Comment:
check this, but i don't think there's any asyncronous stuff happening inside this method, so prob doesn't need to by async